raster/r.proj/testsuite/test_r_proj.py

```diff
@@ -0,0 +1,302 @@
+                "variance": 411.487637404699,
+                "coeff_var": 18.3735049865636,
+                "sum": 35519745.3918571,
+            },
+            "lanczos": {
+                "n": 321724,
+                "null_cells": 2276,
+                "cells": 324000,
+                "min": 56.0589294433594,
+                "max": 156.221099853516,
+                "range": 100.162170410156,
+                "mean": 110.404477059461,
+                "mean_of_abs": 110.404477059461,
+                "stddev": 20.2851957283322,
+                "variance": 411.489165736748,
+                "coeff_var": 18.373526390064,
+                "sum": 35519769.977478,
+            },
+            "bilinear_f": {
+                "n": 324000,
+                "null_cells": 0,
+                "cells": 324000,
+                "min": 55.9677429199219,
+                "max": 156.230346679688,
+                "range": 100.262603759766,
+                "mean": 110.375306993838,
+                "mean_of_abs": 110.375306993838,
+                "stddev": 20.3113089385128,
+                "variance": 412.54927079571,
+                "coeff_var": 18.4020407206177,
+                "sum": 35761599.4660034,
+            },
+            "bicubic_f": {
+                "n": 324000,
+                "null_cells": 0,
+                "cells": 324000,
+                "min": 55.9677429199219,
+                "max": 156.223129272461,
+                "range": 100.255386352539,
+                "mean": 110.375287905093,
+                "mean_of_abs": 110.375287905093,
+                "stddev": 20.3150368008109,
+                "variance": 412.700720218302,
+                "coeff_var": 18.4054213460164,
+                "sum": 35761593.28125,
+            },
+            "lanczos_f": {
+                "n": 324000,
+                "null_cells": 0,
+                "cells": 324000,
+                "min": 55.9677429199219,
+                "max": 156.221099853516,
+                "range": 100.253356933594,
+                "mean": 110.375363786639,
+                "mean_of_abs": 110.375363786639,
+                "stddev": 20.3150742610671,
+                "variance": 412.702242232673,
+                "coeff_var": 18.4054426315072,
+                "sum": 35761617.8668709,
+            },
+        },
+    }
+
+    @classmethod
+    def setUpClass(cls):
+        cls.use_temp_region()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.del_temp_region()
+        if cls.to_clear:
+            cls.runModule(
+                "g.remove", flags="f", type="raster", name=",".join(cls.to_clear)
+            )
+
+    def test_same_resolution(self):
+        self.runModule("g.region", raster="elevation")
+        for method, reference in self.references["same"].items():
+            output_file = "elevation_same_{}".format(method)
+            self.to_clear.append(output_file)
+            self.runModule(
```

Comment:
`assertModule` is better in this context as this tells the framework that this is the module you are testing. It may improve the error messages you (don't) get in the CI.